src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -729,19 +918,19 @@ export default class OrgPerson extends Mixins(CommonMixin, OrgPersonMixin) {
     })
   }
 
-  /**
-   * Returns a new data object from current local properties.
-   */
+  /** Returns a new data object from current local properties. */
   private addPerson (): OrgPersonIF {
-    let person: OrgPersonIF = cloneDeep(this.orgPerson)
```

Comment:
1. don't need to declare type because it's inferred from the result of cloneDeep
2. doesn't need to be editable (let vs const) because we don't reassign person, we change its properties